torch/_torch_docs.py

```diff
@@ -662,6 +706,28 @@ def merge_dicts(*dicts):
     tensor([ 0.9833,  0.0811, -1.9743, -1.4151])
 """.format(**common_args))
 
+add_docstr(torch.atanh,
+           r"""
+atanh(input, out=None) -> Tensor
+
+Returns a new tensor with arctanh (inverse hyperbolic tangent) of the elements of :attr:`input`.
```

Comment:
Same language comment as above. A second sentence describing the allowed inputs would be good.

Reply:
Thanks, @mruberry! This should be resolved here: https://github.com/pytorch/pytorch/pull/38388/commits/234df817a34e6d3450b9274fb27c13aa4b80b278.